docs/programming-oak.md

```diff
@@ -182,8 +258,8 @@ The autogenerated code includes three parts, described in more detail below:
 
 The first part is a trait definition that includes a method for each of the
 methods in the gRPC service, taking the relevant (auto-generated) request and
-response types. The Oak Node implements the gRPC service by implementing this
-trait.
+response types. The Oak Node handler struct implements the gRPC service by
+implementing this trait.
```

Comment:
Might now be worth an xref back to the section above `[as described previously](#grpc-service-handler-node)`

Reply:
Done.